### PR TITLE
Govcon-sandbox-  04 beyond: Remove styles to be added by users1

### PR DESCRIPTION
**Pull request by mahoneycm**
# Summary

Removes styles that the instructions tell users to create. 

Also adds hero component file destination to readme in place of existing `tk`.

>[!NOTE]
> We should maybe add the readme change to `main` as well

### CODE REVIEW

File: src/_includes/hero.html [hunk 1]
```diff
@@ -11,7 +11,7 @@ <h1 class="usa-hero__heading">
         Some short explanatory text. You don’t need more than a couple of
         sentences.
       </p>
-      <a class="dgc-button usa-button" href="javascript:void(0)">Call to action</a>
+      <a class="usa-button" href="javascript:void(0)">Call to action</a>
```
Comment (mejiaj) [inline src/_includes/hero.html:14]: We'll need the `dgc-button` and `dgc-button--callout` class for the exercise. Doesn't seem big enough to have users add it themselves. 

Do you think we should have them modify?

**Review by mejiaj — COMMENTED** [review]
Just a question on template, otherwise LGTM.

File: src/_includes/hero.html (hunk 1 above)
Comment (mahoneycm) [reply src/_includes/hero.html:14]: Currently we are instructing users to add it themselves

> 1. Go to hero component ./src/_includes/hero.html.
>2.  Add the following classes: dgc-button dgc-button--callout.

https://github.com/mejiaj/govcon-sandbox/blob/04-beyond/README.md#extending-button

I only made the change to have it match instructions but we can leave if you think that's better!

Comment (mejiaj) [reply src/_includes/hero.html:14]: Good point, thanks.

**Review by mejiaj — APPROVED** [review]

**Pull request merged**